draft-ietf-httpbis-digest-headers.md

```diff
@@ -112,100 +110,87 @@ The source code and issues list for this draft can be found at
-representation of a resource eg. Range Requests (see Section 14.2 of
+representation of a resource, for example Range Requests (see Section 14.2 of
 {{SEMANTICS}}).
 
-This document replaces [RFC3230] to better align with {{SEMANTICS}} and to
-provide more detailed description of `Digest` usage in request and response
-cases. Changes are intended to be semantically compatible with existing
+Historically, the Content-MD5 header field provided an HTTP integrity mechanism
+but HTTP/1.1 ([RFC7231], Appendix B) obsoleted it due to inconsistent handling
+of partial responses. [RFC3230] defined the concept of "instance" digests and a
+more flexible integrity scheme to help address issues with Content-MD5. It first
+introduced the `Digest` and `Want-Digest` fields. HTTP terminology has evolved
+since [RFC3230] was published. The concept of "instance" has been superseded by
+`selected representation`.
+
+This document replaces [RFC3230]. The `Digest` and `Want-Digest` field
+definitions are updated to align with the terms and notational conventions in
+{{!SEMANTICS}}. Changes are intended to be semantically compatible with existing
 implementations but note that negotiation of `Content-MD5` is deprecated
 {{deprecate-contentMD5}}, `Digest` field parameters are obsoleted
 {{obsolete-parameters}}, "md5" and "sha" digest-algorithms are obsoleted
-{{broken-algorithms}} and the "adler32" algorithm is deprecated
+{{broken-algorithms}}, and the "adler32" algorithm is deprecated
 {{deprecated-algorithms}}.
 
-The value of `Digest` is calculated on selected representation, which is tied to
-the value contained in any `Content-Encoding` or `Content-Type` header fields.
-Therefore, a given resource may have multiple different digest values.
-
-To allow both parties to exchange a Digest of a representation with no content
-codings (see Section 8.4.1 of {{SEMANTICS}}) two more digest-algorithms
-are added ("id-sha-256" and "id-sha-512").
-
-## Goals
-
-The goals of this proposal are:
-
-   1. Digest coverage for either the resource's `representation data` or
-      `selected representation data` communicated via HTTP.
-
-   2. Support for multiple digest-algorithms.
-
-   3. Negotiation of the use of digests.
-
-The goals do not include:
-
-  HTTP message integrity:
-  : Digest mechanisms do not cover the full HTTP message
-    nor its semantic, as representation metadata is not included in the
-    checksum.
-
-  HTTP field integrity:
-  : Digest mechanisms cover only representation and selected
-    representation data, and do not protect the integrity of associated
-    representation metadata or other message fields.
-
-  Authentication:
-  : Digest mechanisms do not support authentication
-    of the source of a digest, message or anything else. These mechanisms,
-    therefore, are not a sufficient defense against many kinds of malicious
-    attacks.
+Calculating the value of `Digest` using selected representation means it is tied
```

Comment:
While I agree in using a single type of list, I think that removing lists at all makes this section less readable.